scripts/simplify_network.py

```diff
@@ -300,7 +300,13 @@ def split_links(nodes):
                 while m not in (supernodes | seen):
                     seen.add(m)
                     for m2, ls in G.adj[m].items():
-                        if m2 in seen or m2 == u:
+                        # there may be AC lines which connect ends of DC chains
+                        ls_is_ac = (
+                            n.lines.loc[n.lines.index == list(ls)[0][1]].tag_frequency
+                            != 0
+                        )
+                        # in case ls is a link, all() on empty series gives True
+                        if m2 in seen or m2 == u or ls_is_ac.all():
```

Comment:
the assumption all on empty returns true may be dangerous for future release of pandas; I'd add an explicit check, shouldn't harm.
Then ready to merge :)

Reply:
Apart of being dangerous, that is also not very clear. Which has actually lead to a mistake which I introduced: in case of empty series (if there are no AC lines, this if should be by-assed which means the condition should be false, not true)  

Thank for double-checking! :) Revised